packages/config/config/devices/0x0129/cp2-5-3_1.78.json

```diff
@@ -43,32 +43,34 @@
 			]
 		},
 		"2": {
-			"label": "Auto Relock On/Off",
+			"label": "Auto Relock",
 			"valueSize": 1,
-			"minValue": -128,
+			"minValue": 128,
 			"maxValue": 0,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Auto Relock On",
-					"value": -128
+					"label": "Enable",
+					"value": 128
 				},
 				{
-					"label": "Auto Relock Off",
+					"label": "Disable",
 					"value": 0
 				}
```

Comment:
What does OpenHAB (or the manual text) have here? -128/0 seems suspicous.

Reply:
I can't actually find any manual with this in it, as it is a module that goes inside other locks, but the others are consistently 0/255 so I'm just going to go with that.